web/api/v6/report_server.thrift

```diff
@@ -319,7 +332,8 @@ service codeCheckerDBAccess {
   // PERMISSION: PRODUCT_ACCESS
   RunDataList getRunData(1: RunFilter runFilter,
                          2: optional i64 limit,
-                         3: optional i64 offset)
+                         3: optional i64 offset,
+                         4: optional list<RunSortMode> sortTypes)
```

Comment:
Do we need a list of RunSortMode? Is it possible to create such an api call on the UI?
I see only one runSortMode element in all of the tests.

Reply:
We are using the same syntax at the `getRunResults` API function:
https://github.com/Ericsson/codechecker/blob/f37829be17d096f22cfbaafcaee11107e41f5049/web/api/v6/report_server.thrift#L366-L372
To be consistent with this I created the same syntax.

Reply:
I get that this way it is consistent but do we actually need it to be a list?
I'm not sure we use multiple values to sort by in the other case.
Where is it used, on the UI I can sort only selecting only one sort mode.
On the CLI I did not found any sort option here `CodeChecker cmd results --help`